hathor/metrics.py

```diff
@@ -27,6 +28,7 @@
 from hathor.transaction.storage import TransactionStorage
 from hathor.transaction.storage.cache_storage import TransactionCacheStorage
 from hathor.transaction.storage.memory_storage import TransactionMemoryStorage
+from hathor.transaction.storage.rocksdb_storage import TransactionRocksDBStorage
```

Comment:
This will break tests on Windows because the rocksdb storage can't be imported. Usually we import only when needed inside functions that will not run in Windows. I see that the way it is being used within an `isinstance` will not work on Windows. Maybe it would be better to have an abstract function in `TransactionStorage` that `TransactionRocksDBStorage` implements, so that the rocksdb specific portions aren't leaked. What do you think?

Reply:
Thanks for the tip. I've checked the error logs, and actually this import was not the problem, but the initialization of this class that I was doing in 2 of my tests.

Skipping these tests in Windows seems to have fixes the problem.

Reply:
That seems good then.